assets/queries/terraform/aws/s3_bucket_public_acl_overridden_by_public_access_block/test/negative2.tf

```diff
@@ -0,0 +1,39 @@
+resource "aws_s3_bucket" "public-bucket3" {
+  bucket = "bucket-with-public-acl-32"
+  acl = "public-read-write"
+}
+
+
+module "s3_bucket" {
```

Comment:
[create_bucket defaults to true](https://github.com/terraform-aws-modules/terraform-aws-s3-bucket/blob/master/variables.tf#L4)
If you're using this module to just update the permissions of the bucket created in a different resource you should set `create_bucket = false`